.github/workflows/release.yml

```diff
@@ -0,0 +1,35 @@
+name: Release
+
+on: [push]
+
+jobs:
+  release:
+    runs-on: ubuntu-latest
+    if: "!contains(github.event.head_commit.message, 'ci skip') && !contains(github.event.head_commit.message, 'skip ci')"
```

Comment:
did you copy pasted it from some other place ? `ci skip` and `skip ci` seems something done for backwards compatibility. maybe one value is enough ?

Reply:
Yes I took the code from the Auto documentation. I think it's because bots 🤖 can use one or the other